test/mlir/conversion/onnx_to_krnl/Math/Elementwise.mlir

```diff
@@ -901,6 +901,99 @@ func.func @test_isnan(%arg0 : tensor<2x3x4xf32>) -> tensor<*xi1> {
 
 // -----
 
+func.func @test_gelu_none(%arg0 : tensor<2x3x4xf32>) -> tensor<2x3x4xf32> {
```

Comment:
If you don't mind, I would suggest to add this test to the related `Elementwise_with_canonicalize.mlir` file, that way it's less sensitive to the redundant operations that are generated, e.g. there are multiple instances of the constant `2`... all that is cleaned up with the additional canonicalized pass.

Reply:
Okay sure! So remove this test from Elementwise.mlir and place it in Elementwise_with_canonicalize.mlir? Or have these two tests in both locations? Just making sure I am following here.

Reply:
Okay fixed!